See [[Patch Notes]] for this wiki's collection of classic EverQuest patch notes.  This page exists to list sources for further patch research.

== Allakazaam Condensed Patch History ==

;[http://everquest.allakhazam.com/history/patches-1999.html Patches] from April 1999 - December 1999
<small>
* Original patch information (April)
* PvP Melee/Caster balance (Sept. 13)
* Rez timers set (Sept. 28)
* Temple of Solusek Ro added as an optional zone (Oct. 13)
* Journeyman Boots become quested (Oct. 13)


;[http://everquest.allakhazam.com/history/patches-2000-1.html Patches] from January 2000 - June 2000

* Plane of Sky is introduced (Jan.)
* Casters respawn with no mana after being killed by a player (Feb. 3)
* City of Paineel is introduced (Feb. 17)
* Erudites can now be Shadowknights (Feb. 17)
* Rogues get the Evade ability (Feb. 17)
* Deity-specific items are introduced (Apr. 10)
* Kerra Ridge and Splitpaw are revamped to higher level (Apr. 10)
* Kunark goes live with a new level cap of 60 (Apr. 21)
* Newbie guards get buffed to their current levels (May)
* The Hole is introduced (June 22)
* Lady Vox and Lord Nagafen are no longer attackable by players over 52nd level


;[http://everquest.allakhazam.com/history/patches-2000-2.html Patches] from July 2000 - December 2000

* NPCs stop saying "hit by non-melee for XX points" (July 19)
* /hidecorpse and /fastdrop commands introduced (July 19 & Aug. 1)
* Wurmslayer becomes a one*hand slasher (Aug. 9)
* NO RENT tags are added to NO RENT items (Aug. 9)
* Epic quests are introduced (Sept. 19)
* Players are now able to exceed 200 in one tradeskill (Sept. 19)
* Corpses of NPCs 55th level or higher get 30-minute decay timers (Sept. 28)
* Pauses are added to socials, /camp command added (Sept. 28)
* Velious goes live (Dec. 6)
* NPCs gain RAMPAGE and FLURRY abilities (Velious Release)


;[http://everquest.allakhazam.com/history/patches-2001-1.html Patches] from January 2001 - June 2001

* /lootnodrop command added (Jan. 9)
* Root and Snare are now seperate effects (Jan. 9)
* NPC Aggro is significantly retuned (Feb. 21)
* Rangers can now sort and filter tracking system (Feb. 21)
* Taunt now affects NPCs above the player's level (Mar. 6)
* EQGems is introduced (Mar. 6)
* The Warrens is introduced (Mar. 14)
* Hate and Fear planes receive revamps: New minibosses, better god loot (Mar. 14)
* Food items get durations (Apr. 17)
* Characters now say "Hail Playername" instead of just "Hail" (Apr. 17)
* /charinfo command introduced (Apr. 17)
* NO RENT items adopt the 30-minute rule (Apr. 17)
* Players no longer get stunned while invulnerable (May 8)
* Items can be moved inside containers while in bank (May 8)
* "Regards you as an ally" is added as a new /consider (May 8)
* Runnyeye revamp (May 30)


;[http://everquest.allakhazam.com/history/patches-2001-2.html Patches] from July 2001 - December 2001

* Spider and Spiderling Silk become stackable (July 10)
* Folded Sheets and regular Sheets of Metal become stackable (July 10)
* Stonebrunt goes live (Aug. 29)
* Server population counts become hidden (Sept. 11)
* Chardok revamp (Oct. 8)
* Graphic upgrades are announced for Luclin (Oct. 5)
* The Firiona Vie (roleplay) Server goes live (Oct. 8)
* Flayed Barbarian Hide Mask gets Illusion: Barbarian added to it (Oct. 8)
* High level NPCs develop immunities to certain spell types, and are no longer outright magic immune (Oct. 8)
* The short-lived "Project M" is introduced, where players can spawn as monsters (Nov. 7)
* /targetgroupbuff is introduced (Nov. 7)
* "Light Blue" /consider is introduced (Nov. 7)
* Luclin goes live (Dec. 4)
</small>

==Baldio Patch History==
*http://www.tski.co.jp/baldio/patch/19992002.php

== Waring McMarrin's Blog Patch Note History ==
* http://www.waringmcmarrin.com/?page_id=43

==Samanna's Reference Desk: Chronology==
* http://samanna.net/eq.general/chronology.shtml

==Neriak.com News==
*http://web.archive.org/web/20010926095411/http://www.neriak.com/

== Aeolwind Timeline ==
[https://www.project1999.com/forums/showthread.php?t=1042&highlight=timeline&page=2]

Note this is '''not''' the official Timeline, according to Nilbog [http://www.project1999.com/forums/printthread.php?t=31014 ref]

"Here is the ORIGINAL progression, we're adhering to this as best we can."

<small>
=== Release (March 16, 1999) ===

*open plane of fear
*remove lustrous russet , amygdalan warriors
*remove cryosilk robe , spinechiller spiders
*remove bladestorm

*open plane of hate

*remove Thex Dagger - original silent watch shield quest reward

*remove box of abu kar, return marfen binkdurple prekunark

SEPTEMBER 13, 1999
*High level pets (44th and 49th lvl spells) do less damage.

OCTOBER 13, 1999
*open temple of solusek ro
*open Plane of Sky
*remove rubicite from ct, replace with ravenscale
*remove guise of the deceiver from assassin, replace with mask of deception
*remove manastone, evil eye
*remove Scimitar of the Mistwalker, vox
*remove jboots from najena, readd hasten and AC

DECEMBER 1, 1999
*Summon Waterstone (Magician) has been added to the game.

DECEMBER 14, 1999
*The new Necromancer spell, Summon Corpse, is now live and functioning as intended.

FEBRUARY 3, 2000
*All spells that grant levitation will no longer function in the Plane of Sky.

FEBRUARY 17, 2000
*nerf mistwalker pet to dissipate after 1 round of combat
*open paineel (we starting with it open i think? )

APRIL 10, 2000
*The Terror Mask will now cast "Fear" instead of "Call of Bones" as previously announced. The "Call of Bones" effect was placed on this item by mistake.

*staff of forbidden rites - needs to be changed to ultra rare at some point

''' To Unlock Kunark: '''
*Lord Nagafen
*Lady Vox
*Phinigel Autropos
*Cazic Thule
*Innoruuk
*Noble Dojorn

===Kunark (April 24, 2000)===

*Reinstate Burning Rapier to updated stats. 7/20 effect:ignite

APRIL 28, 2000
*Due to a feature within the zone, players can no longer bind within Timorous Deep. This functionality was intended in the zone prior to launch, as binding presents a possible balance issue. Players who are already bound in this zone will retain their binding location until they bind someplace else.

MAY 7, 2000
*Gem Encrusted Ring removed from forest loop loottable

MAY 12, 2000
*increase gate guard stats, increase spawn time of city guards from 6 minutes to 20 minutes (nerf guard hunting)

MAY 16, 2000
*Increased the number of frightfingers in the Plane of Fear. Previously only 3 frightfingers would pop per respawn. (haha, already done)

MAY 25, 2000
*The Runed Fighter's Staff, previously usable by Monks and Warriors, has been restricted to Monks only. The statistics on this staff are much more powerful than normally allowed for warriors. This change is retroactive and will be reflected on items currently in inventory. Though we strive to avoid negatively adjusting items after they have been earned, this item presented a balance issue, making the change necessary. We apologize to warriors that currently own this weapon. Runed Fighter's Staff replaced with imbued fighters staff

JUNE 15, 2000
*The Staff of Temperate Flux and the Rod of Insidious Glamour now have a short casting time associated with their activatable effect

JUNE 22, 2000
*The Hole opens
*Vox and Nagafen receive banishment ability

JULY 19, 2000
*nerf deepwater helm effect
*nerf donals vambraces effect
*You can no longer cast invisibility on any pet.

JULY 20, 2000
*/Consent no longer gives other players permission to loot your corpse. Instead, it only gives them the ability to drag your corpse using the /corpse command. This change is due to player request stemming from number of customer service issues raised by some players who were using /consent as a way to steal the belongings of other players. The change will eliminate those concerns and make the game more enjoyable for all of those involved.

AUGUST 9, 2000
*Rangers, Paladins, and SKs now have mana equivalent to that of a pure caster with the same stats. This will give those hybrids a good deal more mana, regardless of their stats

SEPTEMBER 19, 2000
*Mosscovered Twig nerfed into Mosscovered Branch
*Barbed Scale Whip nerfed to offhand only

*EPIC QUESTS - We are pleased to announce that we have implemented new "Fiery Avenger" style quests for every class in the game (including paladins). The ultimate reward for each quest boasts a custom model with unique particle effects. We think that you will be pleased.

NOVEMBER 15, 2000
*Repaired the Forager Bag. It was causing people to lose weight when placing items in the bag. It is supposed to have a weight reduction component, but was not supposed to cause weight loss.

NOVEMBER 29, 2000
*The Monk's hand-to-hand skill cap has been raised to a level equal to their one-handed blunt skill
*remove Locket of Escape
*Rod of Annihilation - rare beyond belief then removed- caretaker, sebilis
*remove Blade of the Black Dragon Eye, replace on loot table as Blade of the Green Dragon Eye
*Circlet of Shadow nerfed to circlet of shadows
*Jade Chokidai Prod - was originally war/rng/shd/brd/rog/shm, nerfed to shmn only

''' To unlock Velious: '''
*Hoshkar
*Venril Sathir
*Talendor
*Gorenaire
*Severilous
</small>

[[Category:Timeline]]